pkg/component/autoscaling/vpa/vpa_test.go

```diff
@@ -184,6 +185,20 @@ var _ = Describe("VPA", func() {
 		Expect(c.Create(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "ca", Namespace: namespace}})).To(Succeed())
 		Expect(c.Create(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "generic-token-kubeconfig", Namespace: namespace}})).To(Succeed())
 
+		vpaFor = func(clusterType component.ClusterType, IsGardenCluster bool) component.DeployWaiter {
```

Comment:
```suggestion
		vpaFor = func(clusterType component.ClusterType, isGardenCluster bool) component.DeployWaiter {
```

I think lowercase is preferable in function arguments.

Reply:
Thanks for catching this typo, I fixed in my latest updates.